scripts/observer.ts

```diff
@@ -191,17 +190,23 @@ export class ActivityManager {
             existingEntry.activityType == Models.ActivityType.Edit) {
             var existingDate = new Date(existingEntry.activityDate);
             if (activityDate.getTime() - existingDate.getTime() < 10000) {
+                console.log("Onload before save event");
                 return;
             }
         }
 
         if (existingEntry) {
+            // remove the entry from it's current location
+            // since we're going to bring to the top of the list.
             activities.splice(existingEntryIndex, 1);
         }
 
-        activities.splice(1000);
-        activities.push(activity);
+        // put the activity on the top of the array
+        activities.unshift(activity);
```

Comment:
Yes.  I'm fine with it being in a weird state for mseng while people start generating new activities.